app/scripts/lib/manifestFlags.ts

```diff
@@ -9,7 +9,10 @@ export type ManifestFlags = {
     nodeIndex?: number;
     prNumber?: number;
   };
-  doNotForceSentryForThisTest?: boolean;
+  sentry?: {
+    tracesSampleRate?: number;
+    doNotForceSentryForThisTest?: boolean;
```

Comment:
Minor, could we remove the `Sentry` from this property as it's now implied by the parent property?